doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -704,57 +688,33 @@ the conversion in hypotheses :n:`{+ @ident}`.
             Goal 1 + 0 = 1.
             unfold Nat.add.
 
-   .. tacv:: unfold @qualid in @goal_occurrences
-
-      Replaces :n:`@qualid` in hypothesis (or hypotheses) designated
-      by :token:`goal_occurrences` with its definition and replaces
-      the hypothesis with its :math:`\beta`:math:`\iota` normal form.
-
-   .. tacv:: unfold {+, @qualid}
-
-      Replaces :n:`{+, @qualid}` with their definitions and replaces
-      the current goal with its :math:`\beta`:math:`\iota` normal
-      form.
-
-   .. tacv:: unfold {+, @qualid at @occurrences }
-
-      The list :token:`occurrences` specify the occurrences of
-      :n:`@qualid` to be unfolded. Occurrences are located from left
-      to right.
-
       .. exn:: Bad occurrence number of @qualid.
          :undocumented:
 
       .. exn:: @qualid does not occur.
          :undocumented:
 
-   .. tacv:: unfold @string
-
-      If :n:`@string` denotes the discriminating symbol of a notation
-      (e.g. "+") or an expression defining a notation (e.g. `"_ +
-      _"`), and this notation denotes an application whose head symbol
-      is an unfoldable constant, then the tactic unfolds it.
-
-   .. tacv:: unfold @string%@ident
-
-      This is variant of :n:`unfold @string` where :n:`@string` gets
-      its interpretation from the scope bound to the delimiting key
-      :token:`ident` instead of its default interpretation (see
-      :ref:`Localinterpretationrulesfornotations`).
+.. tacn:: fold {+ @one_term } {? @occurrences }
 
-   .. tacv:: unfold {+, {| @qualid | @string{? %@ident } } {? at @occurrences } } {? in @goal_occurrences }
+   .. todo PR: following is wrong because you can have multiple one_terms.  How
+      do multiple one_terms interact with occurrences?? I suppose the one_term is
+      a pattern match (& should be a one_pattern?).  What should this say?
```

Comment:
`fold` only supports the `in` clauses, not the `at` clauses, so there is no issue with occurrence counting.  `one_term` is not a pattern, and should not be `one_pattern`.